src/aap_eda/settings/core.py

```diff
@@ -149,6 +149,22 @@
 
 DEFAULT_REDIS_DB = 0
 
+# ---------------------------------------------------------
+# DISPATCHERD SETTINGS
+# ---------------------------------------------------------
+
+DISPATCHERD_DEFAULT_CHANNEL = "default"
```

Comment:
@Alex-Izquierdo Will this channel name have conflicts with other services like controller should have a eda prefix for this?